index.html

```diff
@@ -0,0 +1,159 @@
+        </div>
+        <ul>
+            <li class="links home-page"><a href="index.html">Home</a></li>
+            <li class="links"><a href="about.html">About</a></li>
+            <li class="links"><a href="#">Program</a></li>
+            <li class="links"><a href="#">Join</a></li>
+            <li class="links"><a href="#">Sponsor</a></li>
+            <li class="links last-link"><a href="#">News</a></li>
+            <li class="links focused"><a href="#">CC Campaign</a></li>
+        </ul>
+     </div>
+</header> 
+<main>
+    <!-- Introduction Section -->
+    <section id="intro" class="intro">
+        <h2>"Hello! Sharing world"</h2>
+        <h1>creative commons global summit 2023</h1>
+        <p class="bio">
+            A joyful celebration believing in the value of openness and sharing,
+            creating a positive change with people from over 80 countries is
+            taking place in October, in Korea.
+          </p>
+          <p class="time">2023.10.15(thu) ~ 16(fri)</p>
+          <p class="address">
+            @ National Museum of Korea, Art Center Nabi and more
+          </p>
+    </section>
+    <!-- Main Program Section -->
+    <section id="main-program">
+        <p class="title">Main Program</p>
+  <hr />
+  <div class="main-program">
+    <div class="lecture">
+        <img src="./assets/logos/p5.png" alt="lecture logo" />
+        <h3>Lecture</h3>
+        <p>
+          listen to the speakers from various countries about the messeges
+          of sharing and opening.
+        </p>
+    </div>
+    <div class="ccexhibition">
+        <img src="./assets/logos/p1.png" alt="cc Exhibition logo" />
+        <h3>CC Exhibition</h3>
+        <p>
+          Appreciate various creations applying CC license of artists,
+          organized from Art Center Nabi.
+        </p>
+      </div>
+      <div class="forum">
+        <img src="./assets/logos/p2.png" alt="Forum logo" />
+        <h3>Forum</h3>
+        <p>
+          Have the time to share your thoughts and opinions with experts for
+          each topic.
+        </p>
+      </div>
+      <div class="workshop">
+        <img src="./assets/logos/p3.png" alt="workshop logo" />
+        <h3>Workshop</h3>
+        <p>
+          try creating your own work using open source license rather than
+          just watching at it.
+        </p>
+      </div>
+      <div class="ccignite">
+        <img src="./assets/logos/p4.png" alt="cc ignite logo" />
+        <h3>CC Ignite</h3>
+        <p>
+          Get opportunities to network with CC affiliates around the world,
+          also after the summit.
+        </p>
+      </div>
+  </div>
+  <p class="desk">See the whole program</p>
+  <p class="mob">Join CC Global Summit 2022</p>
+    </section>
+    <!-- Featured Speakers Section -->
+    <section id="featured-speaker">
+        <div id="speakers">
+          
+        </div>
+
+    </section>
```

Comment:
- [ ] According to the Provided design, the mobile section only shows two speakers and then a more button. the partners and footer on the home page are not visible on mobile screens, I'll suggest you update this. :+1: 

- - Your current design:
![Screenshot from 2023-07-07 11-05-20](https://github.com/MohammadYaser/Module-1-Capstone/assets/107632780/d11a39e2-b9f6-4e7e-9ea6-e7d0cb14ce8d)

 - - Given design:
![Screenshot from 2023-07-07 11-05-42](https://github.com/MohammadYaser/Module-1-Capstone/assets/107632780/211c13a3-5d06-4b36-8bf2-23eab3047057)

Reply:
> * [ ]  According to the Provided design, the mobile section only shows two speakers and then a more button. the partners and footer on the home page are not visible on mobile screens, I'll suggest you update this. +1
> * * Your current design:
>     ![Screenshot from 2023-07-07 11-05-20](https://user-images.githubusercontent.com/107632780/251710796-d11a39e2-b9f6-4e7e-9ea6-e7d0cb14ce8d.png)
> * * Given design:
>     ![Screenshot from 2023-07-07 11-05-42](https://user-images.githubusercontent.com/107632780/251710843-211c13a3-5d06-4b36-8bf2-23eab3047057.png)

- [ ] This required changes by the previous reviewer is yet to be implemented. Kindly attend to it.

Reply:
> * [ ]  According to the Provided design, the mobile section only shows two speakers and then a more button. the partners and footer on the home page are not visible on mobile screens, I'll suggest you update this. +1
> * * Your current design:
>     ![Screenshot from 2023-07-07 11-05-20](https://user-images.githubusercontent.com/107632780/251710796-d11a39e2-b9f6-4e7e-9ea6-e7d0cb14ce8d.png)
> * * Given design:
>     ![Screenshot from 2023-07-07 11-05-42](https://user-images.githubusercontent.com/107632780/251710843-211c13a3-5d06-4b36-8bf2-23eab3047057.png)

- [ ] This requested changes by the previous reviewer is yet to be implemented. Kindly attend to it